conan/cli/commands/config.py

```diff
@@ -132,3 +133,16 @@ def config_show(conan_api, parser, subparser, *args):
     args = parser.parse_args(*args)
 
     return conan_api.config.show(args.pattern)
+
+
+@conan_subcommand()
+def config_clean(conan_api, parser, subparser, *args):
+    """
+    Clean the configuration files in the Conan home folder. (Keeping installed packages)
+    """
+    subparser.add_argument("-c", "--confirm", action='store_true',
```

Comment:
Probably doesn't need confirmation. The confirmation only would make sense if applied as the one in other commands:
```
Do you want to clean "remotes.txt", yes/no?
Do you want to clean "settings.yml" yes/no?
...

So probably just drop the arg, not necessary for this command.